language/move-core/types/src/vm_status.rs

```diff
@@ -113,6 +213,30 @@ impl fmt::Display for VMStatus {
     }
 }
 
+impl fmt::Display for RecordedStatus {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "status ")?;
+        match self {
+            RecordedStatus::Executed => write!(f, "EXECUTED"),
+            RecordedStatus::OutOfGas => write!(f, "OUT_OF_GAS"),
+            RecordedStatus::VerificationError => write!(f, "VERIFICATION_ERROR"),
+            RecordedStatus::DeserializationError => write!(f, "DESERIALIZATION_ERROR"),
+            RecordedStatus::MoveAbort(location, code) => {
+                write!(f, "ABORTED with code {} in {}", code, location)
+            }
+            RecordedStatus::ExecutionFailure {
+                location,
+                function,
+                code_offset,
+            } => write!(
+                f,
+                "EXECUTION_FAILURE at bytecode offset {} in function index {} in {}",
```

Comment:
I would even format this in a more compact way, like
`location::idx[offset]`
if that makes sense. It feels that info is too technical and reading it in words just sounds funny.
But just to say, anything works

Reply:
I feel it feels weird since its not the name of the function, its the offset.